packages/web/components/Ticker.tsx

```diff
@@ -0,0 +1,23 @@
+import { Flex, Image } from '@metafam/ds';
+import MetaGameLogo from 'assets/logo.png';
+import fetch from 'node-fetch';
+import React, { useEffect, useState } from 'react';
+
+export const Ticker: React.FC = () => {
+  const [price, setPrice] = useState(0);
+
+  useEffect(() => {
+    fetch(
+      `https://api.coingecko.com/api/v3/simple/price?ids=metagame&vs_currencies=usd`,
+    )
+      .then((response) => response.json())
+      .then((data) => setPrice(data.metagame.usd));
+  }, [price, setPrice]);
+
+  return (
+    <Flex alignItems="center">
+      <Image src={MetaGameLogo} alt="MetaGameLogo" h="2rem" />
```

Comment:
Should use this prop to resize images instead of hardcoded values: https://chakra-ui.com/docs/data-display/image#size

Reply:
Perfect, thanks for the comments hammad. I will turn those around in a bit.